Update readme

aleksyprok opened the pull request:
Update the READM.md file

aleksyprok committed:
Improved R_GRID and Z_GRID tables

aleksyprok committed:
Revise grid convention and mapping explanations

Updated grid representation and mapping details in README.

aleksyprok committed:
Clarify grid representation in README

aleksyprok committed:
Fix wording in Grid convention section of README

aleksyprok committed:
Refine Grid Convention section in README

Updated the formatting and description of the Grid Convention section in the README.

aleksyprok committed:
Enhance boundary convention section in README

Clarified boundary convention in README and provided example.

aleksyprok committed:
Update Boundary Convention header level

Changed section header for Boundary Convention.

aleksyprok committed:
Update boundary convention explanation in README

Clarified the definition and ordering of variables with the suffix '_LTRB'.

aleksyprok committed:
Revise README with project details and goals

Updated project name and added detailed description of RTGSFit, its purpose, and development status.

aleksyprok committed:
Rename RTGSFit to RT-GSFit and enhance README

Updated project name and improved documentation clarity.

aleksyprok committed:
Fix project name formatting in README

Corrected the project name from 'RTGSFit' to 'RT-GSFit' for consistency.

aleksyprok committed:
Fix duplicate content in README.md

Removed duplicate paragraph in documentation.

aleksyprok committed:
Revise README for clarity and user collaboration

Updated the README to improve clarity and encourage user engagement.

aleksyprok committed:
Clarify deployment details in README

Updated description to clarify code deployment.

aleksyprok committed:
Enhance README for better clarity and organization

Refined documentation to improve clarity and general applicability for other tokamaks.

aleksyprok committed:
Refactor README for better readability

Removed unnecessary line breaks and improved formatting.

aleksyprok committed:
Enhance README with detailed tool capabilities

Expanded the description of RT-GSFit to include its capability of approximating the 2D magnetic field in real-time during plasma pulses.

aleksyprok committed:
Revise README for improved clarity and documentation

Updated README to enhance clarity and documentation focus for RT-GSFit.

aleksyprok committed:
Update figure caption in README.md

aleksyprok committed:
compelted merge

aleksyprok committed:
Added in figure

aleksyprok committed:
changed svg

aleksyprok committed:
Set figure to max width

aleksyprok committed:
Putting the image back to 14757

aleksyprok committed:
Set width to max possible

aleksyprok committed:
Set width to 80%

aleksyprok committed:
added MC coil metal

aleksyprok committed:
update_readme

aleksyprok committed:
Updated readme

aleksyprok committed:
Added quite a lot of stuff

aleksyprok committed:
Added style_guide and to do.

aleksyprok committed:
Use COCOS 13 added

aleksyprok commented on README.md:
````diff
@@ -1,55 +1,147 @@
+make SHOT=0 RUN_NAME=no_mds DEBUG=1
+```
+
+Here, the `SHOT` and `RUN_NAME` parameters may be set to any integer or string, respectively—these values are not required for compilation at this stage and will be explained later. The `DEBUG=1` option enables additional compiler flags that improve diagnostics and debugging support, at the cost of slower runtime performance. This configuration is used in the example provided in the [RT-GSFit vs. ...
````
I like underscores

aleksyprok commented on README.md:
```diff
@@ -1,55 +1,147 @@
+<p align="center">
+  <img src="https://raw.githubusercontent.com/aleksyprok/rtgsfit_media/refs/heads/main/GIFs/analytic_vs_rtgsfit.gif" alt="rtgsfit_vs_analytic" style="width:80%; height:auto;">
+  <br>
+  <em>Figure 3: Convergence of the RT-GSFit poloidal flux and associated flix loop measurements toward the analytic reference solution.</em>
```
Well spotted!

aleksyprok committed:
Fixed spelling errors

FilipJanky commented on README.md:
```diff
@@ -1,55 +1,150 @@
 cd src/
 make clean
-make SHOT=99000230 RUN_NAME=RUN02 DEBUG=1
+make SHOT=99000230 RUN_NAME=RUN12 DEBUG=0
```
I didn't know we are using already run12. I used typically run10

FilipJanky reviewed (approved):
Alex, you can be really proud of yourself. Great work and fantastic delivery!

FilipJankyTE reviewed (approved):
Brilliant work Alex!

aleksyprok committed:
Revise 'To Do' section for clarity and tracking

Updated the 'To Do' section to reflect new tracking methods and improved formatting.

aleksyprok committed:
Improve clarity in GSFit vs RT-GSFit section

Clarify the description of GSFit and RT-GSFit in the README.
